manifests/v/VSDC/Editor/7.1.1.390/VSDC.Editor.locale.en-US.yaml

```
@@ -1,26 +1,26 @@
-# Created with YamlCreate.ps1 v2.1.0 $debug=NVS1.5-1-19041-1320
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
-
-PackageIdentifier: VSDC.Editor
-PackageVersion: 6.9.5.382
-PackageLocale: en-US
-Publisher: Flash-Integro LLC
-# PublisherUrl: 
-# PublisherSupportUrl: 
-PrivacyUrl: https://www.videosoftdev.com/privacy-policy
-# Author: 
-PackageName: VSDC Free Video Editor
-PackageUrl: https://www.videosoftdev.com/free-video-editor
-License: Proprietary
-LicenseUrl: https://www.videosoftdev.com/terms-and-conditions
-Copyright: © 2011-2022, www.videosoftdev.com, Multilab LLC
-# CopyrightUrl: 
-ShortDescription: The video editor features a modern and highly intuitive interface that uses all kinds of wizards to make video creation and editing as simple and hassle-free as possible.
-Description: The video editor is intended for editing video files and creating videos of any complexity involving various visual and audio effects. The program offers rich functionality and yet has a simple and intuitive interface, allowing you to create videos with a bare minimum of efforts
-Moniker: vsdc-editor
-# Tags: 
-# Agreements: 
-# ReleaseNotes: 
-# ReleaseNotesUrl: 
-ManifestType: defaultLocale
-ManifestVersion: 1.1.0
+# Created with YamlCreate.ps1 v2.1.2 $debug=AUSU.5-1-19041-1645
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: VSDC.Editor
+PackageVersion: 7.1.1.390

```

```suggestion
PackageVersion: 7.1.1.392
```
Version got updated.